File: Java/chapter_15/app/src/main/java/com/glella/criminalintent/database/CrimeDbSchema.java
package com.glella.criminalintent.database;

public class CrimeDbSchema {

    public static final class Crimetable {
        public static final String NAME = "crimes"; // Table name

        // Column names
        public static final class Cols {
            public static final String UUID = "uuid";
            public static final String TITLE = "title";
            public static final String DATE = "date";
            public static final String SOLVED = "solved";
            public static final String SUSPECT = "suspect";
        }


    }
}

File: Kotlin/chapter_14_challenge/app/src/main/java/com/glella/criminalintent/database/CrimeDbSchema.kt
package com.glella.criminalintent.database

class CrimeDbSchema {

    object Crimetable {
        val NAME = "crimes" // Table name
        val VERSION = 1
        val DATABASE_NAME = "crimeBase.db"

        // Column names
        object Cols {
            val UUID = "uuid"
            val TITLE = "title"
            val DATE = "date"
            val SOLVED = "solved"
        }


    }
}